MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```
@@ -26,13 +27,23 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 	}
 
 	func display(_ viewModel: FeedLoadingViewModel) {
+		errorView.hideMessage()
```

It is `FeedPresenter` who should hide the `errorView`, please move this logic to the presenter
```suggestion
```
To hide the error when loading starts you can add a call `display(FeedErrorViewModel(errorMessage: nil))` to `FeedPresenter.didStartLoadingFeed`